flake.nix

```diff
@@ -17,34 +16,36 @@
       systems = [ "x86_64-linux" "i686-linux" "x86_64-darwin" "aarch64-linux" ];
 
       forAllSystems = f: lib.genAttrs systems (system: f system);
-
     in
-    {
-      lib = lib // {
-        nixosSystem = { modules, ... } @ args:
-          import ./nixos/lib/eval-config.nix (args // {
-            modules = modules ++
-              [ { system.nixos.versionSuffix =
+      {
+        lib = lib // {
+          nixosSystem = { modules, ... } @ args:
+            import ./nixos/lib/eval-config.nix (args // {
+              modules = modules
+              ++ [
+                {
+                  system.nixos.versionSuffix =
                     ".${lib.substring 0 8 self.lastModified}.${self.shortRev or "dirty"}";
                   system.nixos.revision = lib.mkIf (self ? rev) self.rev;
                 }
               ];
-          });
-      };
+            }
+            );
```

Comment:
these two should be collapsed. I think the rule is that if N containers are opened on the same line, then they should be closed on the same line as well